packages/serverpod/lib/src/server/serverpod.dart

```diff
@@ -212,13 +212,26 @@ class Serverpod {
   /// started.
   late final ClusterManager cluster;
 
+  /// Define CORS headers to allow cross-origin requests.
+  final Map<String, dynamic> headers;
+
   /// Creates a new Serverpod.
   Serverpod(
     List<String> args,
     this.serializationManager,
     this.endpoints, {
     this.authenticationHandler,
     this.healthCheckHandler,
+    this.headers = const {
+      'Access-Control-Allow-Origin':
+          '*', // Allow all origins or specify a specific origin.
+      'Access-Control-Allow-Methods':
+          'GET, POST, PUT, DELETE', // Add allowed methods.
+      'Access-Control-Allow-Headers':
```

Comment:
Does this risk other headers being blocked by firewalls? E.g., forwarding of IP addresses through X-* headers, etc.

Reply:
I think  no impact on custom headers

Reply:
Think or know? I bad decision will impact 100s of users. 😅

Reply:
I performed a test with a custom 'X-*'  header  , and it was not allowed. Our server configuration only permits the 'Content-Type' header.
<img width="464" alt="Screenshot 2023-10-09 at 9 08 20 PM" src="https://github.com/serverpod/serverpod/assets/40236753/899d1dd2-1621-4abb-a1c4-224fbf591ff3">